corehq/apps/hqwebapp/utils/bootstrap/status/bootstrap3_to_5.json

```diff
@@ -46,8 +46,5 @@
   },
   "corehq.tabs": {
     "in_progress": true
-  },
```

Comment:
I think the better solution to this is to have the tests explicitly ignore cloudcare javascript, so that the reasoning for not including this is more obvious with a comment. right now the reasoning is buried in git history.
however, i'm also trying to understand what's bespoke about the cloudcare js that it shouldn't run tests against it? The only thing the consistency tests look for is a reference to a `bootstrap3` path if the js is not in a split directory

Reply:
it's important for tests to still cover templates. but still wondering why ignore if all its checking is a reference to a `bootstrap3` path

Reply:
(resolved offline)